src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Extensions/ProductExtension.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.Linq;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.StoreModule.Core.Model;
+using VirtoCommerce.XDigitalCatalog.Breadcrumbs;
+
+namespace VirtoCommerce.XDigitalCatalog.Extensions
+{
+    public static class ProductExtension
+    {
+        public static IEnumerable<Breadcrumb> GetBreadcrumbs(this CatalogProduct product, Store store, string cultureName)
+        {
+            if (product.Category != null)
```

Comment:
Invert if, please.

Reply:
I can't. ProductBreadcrumb should be calculated in all cases.